server/pypi/packages/scipy/patches/chaquopy.patch

```diff
@@ -325,77 +244,34 @@ diff -aur src-orig/scipy/special/utils/makenpz.py src/scipy/special/utils/makenp
 diff -aur src-orig/scipy/stats/_generate_pyx.py src/scipy/stats/_generate_pyx.py
 --- src-orig/scipy/stats/_generate_pyx.py	2023-09-20 11:08:03.708957161 +0200
 +++ src/scipy/stats/_generate_pyx.py	2023-09-21 13:13:09.463910299 +0200
-@@ -8,9 +8,9 @@
+@@ -8,6 +8,10 @@
      A new random C API was added in 1.18 and became stable in 1.19.
      Prefer the new random C API when building with recent numpy.
      '''
--    import numpy as np
--    ver = tuple(int(num) for num in np.__version__.split('.')[:2])
--    return ver < (1, 19)
-+    # Chaquopy: we force Numpy to 1.20 version so not old
-+    return False
-+    
- 
- 
- def make_biasedurn():
++    # Chaquopy: prevent the compiled parts of NumPy from being imported
++    import builtins
++    builtins.__NUMPY_SETUP__ = True
++
+     import numpy as np
+     ver = tuple(int(num) for num in np.__version__.split('.')[:2])
```

Comment:
Since this is only importing NumPy to check the`__version__`, it doesn't need to be removed.